hasjob/templates/macros.html.jinja2

```diff
@@ -216,3 +237,71 @@
     </script>
   {%- endwith %}
 {%- endmacro -%}
+
+{%- macro conversion_funnel_setup() -%}
```

Comment:
If this function isn't taking/using any server-side generated values, perhaps better for it not be a macro and just be a JS function?

Reply:
Yes, this should be in the app script, not a macro.